Documentation/ApiOverview/Hooks/EventDispatcher/Index.rst

```diff
@@ -26,6 +26,71 @@ Benni Mack: "Don't get hooked, listen to events! PSR-14 within TYPO3 v10" @ TYPO
   Additional background on the implementation can be found at https://usetypo3.com/psr-14-events.html
 
 
+.. _EventDispatcherQuickStart:
+
+Quick start
+===========
+
+.. _EventDispatcherQuickStartDispatching:
+
+Dispatching an event
+--------------------
+
+#. Create an event class. 
+
+   An event class is basically a plain PHP object with getters for immutable
+   properties and setters for mutable properties. It contains a constructor for all properties::
+   
+      class DoingThisAndThatEvent {
```

Comment:
```suggestion
      final class DoingThisAndThatEvent {
```

All new code should be [final by default](https://matthiasnoback.nl/2018/09/final-classes-by-default-why/).